### DISCUSSION TITLE
Does not work on ThinkPad T14 AMD Gen1

### DISCUSSION THREAD

**Discussion by pomoke** [DavidS95/Smokeless_UMAF#41] [Q&A] [answered]
UMAF does not show a working menu on a ThinkPad T14 Gen1 (AMD).

Here's part of message:
```
SetupBrowser
No of Handle - 2
Could not load the image on Handle 0 - Not Found
Image Returned - Success 0
```

**Comment by pomoke** [comment]
Is this a UMAF problem or bug of EFI firmware?

**Comment by SKUndef** [comment]
I have the same problem, it gives 3 of those errors and then black screen.

**Comment by ghost** [comment, marked as answer]
most of Thinkpad bios require more effort to get it to load,
and I didn't spent time fixing/implementing it.

**Comment by ExistedJohn** [comment]
I met this issue, too. Some measures may taken by Lenovo to enhance the bios tamper detection in the newer version of BIOS, the document is here[https://datacentersupport.lenovo.com/ai/nb/product_security/ps500554-thinkpad-bios-vulnerabilities](url)
I`m gonna try to downgrade my BIOS version and try again. My device is X13 GEN3, however, I think it may be practical for each ThinkPad device.

**Reply by ExistedJohn** [reply]
What`s more UEFI Variable protection was mentioned in this document, which means that the existing way of modifying BIOS by editing UEFI variables will be ineffective on ThinkPad devices, that`s a tough one.

**Reply by pmnlla** [reply]
I'm on a Thinkpad T14 Gen 3. The took seems to boot for me on bios version 1.46, but it cannot save changes.

**Reply by Collin8000** [reply]
Still no fix for me, I too, am on a thinkpad t14 gen 1 amd with a pro 4750U, the tamper protection bug isnt present on my thinkpad, do you think there will be any way to fix this in the future?
